practice-one/src/components/service.html

```diff
@@ -0,0 +1,7 @@
+<section class="service-container">
+  <div><img src="./assets/images/service.png" alt="Service's illustration"></div>
+  <div class="service-description-container">
+    <p class="service-header">We are <span style="color: var(--color-primary);">more</span> than <span style="color: var(--color-accent);">multiple</span> service</p>
```

Comment:
I already pointed out in previous PR, do not use inline style

Reply:
thanks, i fixed it